java/src/main/java/org/rocksdb/TickerType.java

```diff
@@ -361,26 +361,6 @@ public enum TickerType {
      */
     GET_UPDATES_SINCE_CALLS((byte) 0x41),
 
-    /**
-     * Miss in the compressed block cache.
-     */
-    BLOCK_CACHE_COMPRESSED_MISS((byte) 0x42),
```

Comment:
Probably wrong for new numbers

Reply:
I don't think it would be wrong, as in portal.h they are just hard coded in the same way. I updated the PR and removed the gap anyway to be safer.